tests/protocol/test_wire.py

```diff
@@ -58,39 +58,52 @@ def _test_getters(self, test_data, __, MockCryptUtil, _):
                                 '4037FBF5F1F3014F99B5D6C7799E9B20E6871CB3.crt')
```

Comment:
The `protocol.get_certs()` call gets the cert, which in turn calls `self.certs = Certificates(self, xml_text)` in `azurelinuxagent.common.protocol.wire.WireClient#get_certs`. And in this constructor, the Certs are parsed.